src/connector.js

```diff
@@ -345,13 +346,7 @@ module.exports = class Connector extends events.EventEmitter {
   _initialise() {
     this.query( this.statements.initDb( this.options.schema ), ( error, result ) => {
       if( error ) {
-        // retry for errors caused by concurrent initialisation
-        if( error.code === INTERNAL_ERROR ) {
-          this._initialise()
-          return
-        } else {
-          throw error
-        }
+        return this._initialise()
```

Comment:
what if this is a legit error ( like the query being invalid ) as opposed to an initialization issue?